packages/docusaurus-plugin-content-docs/src/index.ts

```diff
@@ -175,25 +176,30 @@ export default function pluginContentDocs(
       // Prepare metadata container.
       const docsMetadataRaw: DocsMetadataRaw = {};
       const docsPromises = [];
+      const includeDefaultDocs = !(
+        options.skipNextRelease && process.argv[2] === 'build'
```

Comment:
I think we should include the argv[2] as a LoadContext value

Reply:
also, includeDefaultDocs and skipNextRelease names are not consistent, it's not instantly clear that those 2 names are related to me. 

What about

`const skipNextRelease = options.skipNextRelease && process.argv[2] === 'build';`

(if skipNextRelease is the name we keep)

Reply:
I do not see a contradiction in fact, the `includeDefaultDocs` name is pretty clear. This is the final condition, and `skipNextRelease` (`excludeNextVersionDocs`) is just a plugin option.

Reply:
somehow, it's a name that is the "opposite" of the plugin option.

I'd rather have something like that:

```
const excludeNextVersionDocs = options.excludeNextVersionDocs && process.argv[2] === 'build'

if (!excludeNextVersionDocs) {
  // include next version docs
}
```

probably subjective ;)